### ISSUE TITLE
Do you have any plans to release the MAME emulator on the App Store?

### ISSUE THREAD

**Issue by Alex92908** [yoshisuga/MAME4iOS#457]
@yoshisuga 
Nowadays, emulator apps are allowed to release on the App Store, like iGBA, are you going to release this App on the App Store?

**Comment by yoshisuga** [comment]
Yes! In fact I submitted it for review and got a rejection notice. We’re
trying to resolve this now and hopefully will have an update soon.

**Comment by Alex92908** [comment]
Well done, looking forward to hear good news from you soon~

**Comment by Dai-Makaimura** [comment]
This is a fantastic news, thanks a lot yoshisuga!!! Lookin forward to see Mame4ios on the Apple Store!

**Comment by eduo** [comment]
If it's not a problem to share, what was the reason for the rejection?

**Comment by ultratiem** [comment]
Godspeed guys! 🫶

**Comment by yoshisuga** [comment]
Just another update: we got rejected because of "spam", where they think we submitted code similar to others who may have submitted MAME. 

I sent a reply to App Store review saying that MAME is open source and this is the canonical port for iOS and that I'm the owner of the repository. Hopefully it will be resolved soon.

**Comment by RussellKR** [comment]
Apple removed 'iGBA' 'coz they were copying someone else's work!
Testut Tech is real owner of Delta, and currently, Delta is No:1 downloaded app US Canada...
I think you have to prove that you own this project. 
Thank you @yoshisuga for considering this request.

**Comment by ultratiem** [comment]
Thanks for the update Yoshi. You may also want to reach out to 9to5 Mac, AppleInsider and MacRumors with the news. They will surely cover your story and give you a lot of credibility that you are the owner and the app is legitimate.

Apple also monitors those sites so it may also grease the wheels.

**Comment by eduo** [comment]
> Apple removed 'iGBA' 'coz they were copying someone else's work! Testut Tech is real owner of Delta, and currently, Delta is No:1 downloaded app US Canada... I think you have to prove that you own this project. Thank you @yoshisuga for considering this request.

It's all more complicated than this. "MAME" is not the same as "MAME4iOS". Both "MAME4iOS" and "iGBA" have their source openly available so people will try to publish it themselves.

If the licenses don't allow it then Apple might pull it out or respect the repository's owner to have preference, but it's not as straightforward.

And if the person is posting a MAME port (not MAME4iOS) when it's 100% allowed, because that part is owned by the MAME team and has its own license.

In the long run all these issues will be resolved but at the beginning it's like a gold rush where both well-intentioned people and opportunists will try to be the first. Especially given the success of Delta.

**Comment by yoshisuga** [comment]
> Thanks for the update Yoshi. You may also want to reach out to 9to5 Mac, AppleInsider and MacRumors with the news. They will surely cover your story and give you a lot of credibility that you are the owner and the app is legitimate.
> 
> Apple also monitors those sites so it may also grease the wheels.

That's an appealing idea but I'd like to hold off on notifying media sites about this and try to resolve this through App Store review to get more information, but thanks for suggesting this.

**Comment by DNC-7** [comment]
Hope to see you in AppStore soon

**Comment by benjenq** [comment]
Last year I tried to compile the original code and installed it on the iPhone 5s with iOS 12. It works perfect!  Hope to see MAME4iOS in App Store in the near future.

**Comment by steverae68** [comment]
Oh crikey, so looking forward to this 👍

**Comment by stagelevel** [comment]
Could anyone please provide an update on this? Thanks

**Comment by eduo** [comment]
> > Thanks for the update Yoshi. You may also want to reach out to 9to5 Mac, AppleInsider and MacRumors with the news. They will surely cover your story and give you a lot of credibility that you are the owner and the app is legitimate.
> > Apple also monitors those sites so it may also grease the wheels.
> 
> That's an appealing idea but I'd like to hold off on notifying media sites about this and try to resolve this through App Store review to get more information, but thanks for suggesting this.

Were you ever given any updates on this? I see people recycling messages from April because there's no new information.

Was it possible to appeal to Apple the "spam" moniker? How hard is is even to make them listed to an explanation of it being a false positive?
